mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```diff
@@ -58,7 +58,9 @@ storage:
       contents:
         inline: |
             #!ipxe
-            set initiator-iqn iqn.2023-11.coreos.diskless:testsetup
+            # set some random uuid as the initiator iqn, otherwise
```

Comment:
it's not really correct to say this isn't used anywhere. It will be passed to `/sys/firmware/ibft/` and iscsiadm will use it to connect to the target. 
We don't have ACL filtering on this test so the value doesn't affect anything but it's still used

Reply:
Thanks for the context. I'll update the commit message and the comment in the next upload.